src/app/pages/rpc-send-transfer/use-rpc-send-transfer.ts

```diff
@@ -32,13 +34,14 @@ export function useRpcSendTransfer() {
   if (!origin) throw new Error('Invalid params');
 
   return {
-    address,
-    amount,
+    recipients,
     origin,
     utxos,
+    totalAmount: recipients.reduce((acc, { amount }) => acc + Number(amount), 0),
```

Comment:
```suggestion
    totalAmount: sumNumbers(recipients.map(({ amount }) => amount))
```

uses helper, avoids floating point maths